src/Components/Button/Button.scss

```diff
@@ -0,0 +1,24 @@
+.button{
+    background-color: #0000ff;
+    color: white;
+    width: 254px;
```

Comment:
No static lengths, what if we need other sizes of buttons? Should wrap to content